### PR TITLE
Peer Channel details href should show the fully qualified domain name…

### PR DESCRIPTION
**Pull request by AustEcon**
… (FQDN)

- Otherwise, if for example the EXTERNAL_HOST=0.0.0.0, the peer channel details will report the url as something like this: `http://0.0.0.0:47124/api/v1/channel/oLwQpplxAvQxOF9UoQO3BTeJylRauGcVlmPrlPqlDiHx8xVRlDdj6VGMY1dSURncEjyL5m2cEY_p0N1NbBqbiA==`.
- It should be x.bitcoinx.io instead of 0.0.0.0

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,8 @@
-EXTERNAL_SERVER_HOST=127.0.0.1
-EXTERNAL_SERVER_PORT=47124
-INTERNAL_SERVER_HOST=127.0.0.1
-INTERNAL_SERVER_PORT=47126
+EXTERNAL_HOST=127.0.0.1
+EXTERNAL_PORT=47124
+EXTERNAL_FQDN=x.bitcoinx.io
```
Comment (rt121212121) [inline .env:3]: This needs to be 127.0.01 in the repo, otherwise it breaks things?

Comment (AustEcon) [reply .env:3]: Okay. Thanks

Comment (rt121212121) [reply .env:3]: It looks like you pushed the wrong file for the public repo.

**Review by rt121212121 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (AustEcon) [reply .env:4]: Aha... I've pushed more updates... The reading from the .env file was happening after the internal_host, external_fqdn and external_host had already been set... Have now moved it to happen earlier.

Comment (rt121212121) [reply .env:4]: I don't understand. This is still the same. What does it have to do with regtest?

**Review by rt121212121 — CHANGES_REQUESTED** [review]

**Review by rt121212121 — APPROVED** [review]

**Pull request merged**